website/sidebars.json

```diff
@@ -2,49 +2,53 @@
   "docs": {
     "About Cumulus": [
       "cumulus-docs-readme",
+      "getting-started",
       "architecture",
       "interfaces",
       "glossary",
       "team"
     ],
-    "Cumulus Workflows": [
+    "Overviews": [
       "workflows/workflows-readme",
       "workflows/protocol",
       "workflows/input_output",
       "workflows/cumulus-task-message-flow",
+      "workflows/workflow-triggers"
+    ],
+    "Deployment": [
+      "deployment/deployment-readme",
+      "deployment/terraform-best-practices",
+      "deployment/upgrade-readme",
+      "deployment/components",
+      "deployment/create_bucket",
+      "deployment/thin_egress_app",
+      "additional-deployment-options/api-gateway-logging",
```

Comment:
I'm on board with a lot of this reorganization, but would it make sense to actually move the files to match their new place in the menu? So move `additional-deployment-options/api-gateway-logging` to `deployment/api-gateway-logging` for example. It might be a bit confusing editing the docs otherwise

Reply:
Yes, the reason I did the mapping first as a draft PR is because I didn't want to go through all of that work if you hated the reorganization and then have to redo it.

Reply:
I made these updates.